chalicelib/checks/wrangler_checks.py

```diff
@@ -1099,3 +1099,103 @@ def share_core_project(connection, **kwargs):
     action.output = action_logs
     return action
 
+@check_function()
+def update_variant_genelist(connection, **kwargs):
+    """
+    Searches for variant samples with genes in gene lists that are not
+    currently embedded in the item. 
+
+    Because of reverse link from gene to gene list, variant samples are not
+    invalidated upon addition of new gene list. This check and the associated
+    action search through variant samples with genes belonging to all current
+    gene lists and add them to the indexing queue if the gene lists are not
+    embedded.
+    """
+
+    check = CheckResult(connection, 'update_variant_genelist')
+    genelist_search = ff_utils.search_metadata(
+        'search/?type=GeneList&field=uuid&field=genes.uuid', 
+        key=connection.ff_keys
+    )
```

Comment:
instead of doing this for every genelist, i think you could only search newly added ones, especially if the check would run every day. maybe you could add a kwarg for days that by default is 1, and then the search url adds `&date_created=<today's date - #days>`

Reply:
This is a good idea. I implemented it with `last_modified.date_modified` rather than `date_created` since the gene lists can be overwritten.

Reply:
that's fine if the gene lists always get created via a post + patch - some items don't have a last_modified field if they haven't been edited (but all the gene lists on prod seem to have it)

Reply:
I noticed the ones created by the submission UI on cgap-test did not, so I'm catching both date_created and last_modified within the time frame. Kent's submitted_by update on dev seems to always create a last_modified field, so perhaps all future gene lists will have the field as well, but a little redundancy won't hurt.